src/astro/sql/operators/agnostic_save_file.py

```diff
@@ -79,13 +80,13 @@ def convert_sql_table_to_dataframe(
         hook_kwargs = {
             Database.POSTGRES: {
                 "postgres_conn_id": input_table.conn_id,
-                "schema": input_table.database,
+                "schema": getattr(input_table.metadata, "database", None),
```

Comment:
The Metadata on the table already has the default of `None` so we don't need a getattr here. We can just do `input_table.metadata.database`

https://github.com/astro-projects/astro/blob/00620d06ee3e9452cab1c82a53d430c6b8358867/src/astro/sql/tables.py#L22